server/src/main/java/org/elasticsearch/index/query/QueryStringQueryBuilder.java

```diff
@@ -97,13 +91,7 @@
     private static final ParseField MINIMUM_SHOULD_MATCH_FIELD = new ParseField("minimum_should_match");
     private static final ParseField QUOTE_FIELD_SUFFIX_FIELD = new ParseField("quote_field_suffix");
     private static final ParseField LENIENT_FIELD = new ParseField("lenient");
-    private static final ParseField LOCALE_FIELD = new ParseField("locale")
-            .withAllDeprecated("Decision is now made by the analyzer");
```

Comment:
I think we should add this to the breaking changes doc above?

Reply:
Done, thanks